lighthouse-core/test/lib/style-helpers-test.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const StyleHelpers = require('../../lib/styles-helpers');
+const assert = require('assert');
+
+const stylesheets = require('../fixtures/styles-gatherer.json');
+
+/* eslint-env mocha */
+
+describe('style helpers', () => {
+  describe('filterStylesheetsByUsage()', function() {
+    it('can take optional arguments', () => {
+      let results = StyleHelpers.filterStylesheetsByUsage(stylesheets);
+      assert.equal(results.length, 0);
+
+      results = StyleHelpers.filterStylesheetsByUsage(
+          stylesheets, 'display', 'box');
+      assert.equal(results.length, 1, 'accepts CSS property name/value pair');
+
+      results = StyleHelpers.filterStylesheetsByUsage(
+          stylesheets, 'display');
```

Comment:
maybe change this to a property name that doesn't work with the value below, just to ensure that `filterStylesheetsByUsage` isn't changing the original stylesheets? Maybe use `color` here, so it's filtered only for `color` entries, for example, but keep `box` below, which will never be the value for a `color` property?